config/environment.rb

```diff
@@ -8,22 +8,30 @@
 require 'sinatra/activerecord'
 require 'sinatra/contrib/all' # Requires cookies, among other things
 
-require 'pry'
 
+
+ENV['RACK_ENV'] ||= 'development'
```

Comment:
I actually noticed this in docs for Sinatra. I wonder if it is worth commenting that Sinatra basically does that someplace. Many questions result from: "but where does it find out if it's development or production?"